torch/csrc/utils/pybind.cpp

```diff
@@ -79,5 +79,26 @@ py::handle type_caster<c10::SymFloat>::cast(
   }
 }
 
+bool type_caster<c10::Scalar>::load(py::handle src, bool) {
+  TORCH_INTERNAL_ASSERT(0, "pybind11 loading for c10::Scalar NYI (file a bug if you need it)");
+}
+
+py::handle type_caster<c10::Scalar>::cast(
+    c10::Scalar scalar,
```

Comment:
```suggestion
    const c10::Scalar & scalar,
```
Doesn't seem to need to take ownership?